### DISCUSSION TITLE
Ukrainian language

### DISCUSSION THREAD

**Discussion by mmospanenko** [outline/outline#5710] [Ideas]
I saw in last release you removed Russian language, we appreciate this

Will be great to see Ukrainian in the supported languages list

Thank you

**Comment by tommoor** [comment]
Ukrainian language support will be in the next release, the coverage is almost 100%

https://translate.getoutline.com/translate/outline/44/en-uk?filter=basic&value=3

**Reply by inkloop** [reply]
Hello, I added missing pieces - 100% ⭐️ Can't wait Ukrainian language. If I can do anything else to make sure it's going live ASAP, please let us know. Thank you 🙏🏻

**Comment by AaronHuston** [comment]
Isn't it a manifestation of nazism to judge people as native speakers of the Russian language? It will be funny to watch your government attack someone tomorrow and forget to ask you. 🤔

**Reply by inkloop** [reply]
The strength of a nation lies in its citizens' willingness to take ownership of their government and shape its destiny. The responsibility for a government's actions rests with its citizens. When they don't take ownership, accountability wanes.

FYI, I think that your question is out of topic, everyone knows that "good russian - **** russian"
